pkg/client/convert.go

```diff
@@ -0,0 +1,53 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/goharbor/acceleration-service/pkg/task"
+	"github.com/goharbor/harbor/src/controller/event"
+	"github.com/goharbor/harbor/src/pkg/notifier/model"
+	"github.com/pkg/errors"
+)
+
+func (client *Client) CreateTask(src string, sync bool) error {
+	res := model.Resource{}
+	res.ResourceURL = src
+	eventData := model.EventData{}
+	eventData.Resources = []*model.Resource{&res}
+	payload := new(model.Payload)
+	payload.Type = event.TopicPushArtifact
+	payload.EventData = &eventData
```

Comment:
Could we write in declarative way?
Something like: 

(not tested)
```golang
payload := &model.Payload{
	Type: event.TopicPushArtifact,
	EventData: &model.EventData{
		Resources: []*model.Resource{
			{
				ResourceURL: src,
			},
		},
	},
}
```

Reply:
Fixed